### ISSUE TITLE
There is no way to get cluster information through ClusterAPI

### ISSUE THREAD

**Issue by zhongchun** [mars-project/mars#2617]
There is no method to get cluster information when you want to know the configs, worker resources of the cluster. Especially when the cluster is autoscaled, the cluster information changes.

**Comment by hekaisheng** [comment]
> We already have `get_nodes_info` API to query all information needed for nodes.

It's about the API implemented in #2602.

**Comment by wjsi** [comment]
> > We already have `get_nodes_info` API to query all information needed for nodes.
> 
> It's about the API implemented in #2602.

Yeah, but if we already have a usable API why do we need another.

**Comment by hekaisheng** [comment]
> > > We already have `get_nodes_info` API to query all information needed for nodes.
> > 
> > 
> > It's about the API implemented in #2602.
> 
> Yeah, but if we already have a usable API why do we need another. These two APIs overlap with each other.

So is `get_nodes_info` API enough for Ray backend? @zhongchun

**Comment by zhongchun** [comment]
> > > > We already have `get_nodes_info` API to query all information needed for nodes.
> > > 
> > > 
> > > It's about the API implemented in #2602.
> > 
> > 
> > Yeah, but if we already have a usable API why do we need another. These two APIs overlap with each other.
> 
> So is `get_nodes_info` API enough for Ray backend? @zhongchun

Also needs other information, like configs. So `get_cluster_info` maybe better.

**Comment by wjsi** [comment]
Just recording these missing information into node info will suffice. No need to add another API to introduce risks for future maintainence.
